src/components/SelectedRecordsModal/SelectedRecordsModal.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { noop } from 'lodash';
+
+import {
+  Modal,
+  ModalFooter,
+  Button,
+  MultiColumnList
+} from '@folio/stripes/components';
+
+const SelectedRecordsModal = ({
+  isOpen,
+  selectedRecords,
+  columnMapping,
+  formatter,
+  onCancel,
+}) => {
+  const intl = useIntl();
```

Comment:
Can we use FormattedMessage component instead? We usually use them whenever possible.